nucypher/policy/conditions/evm.py

```diff
@@ -102,6 +91,13 @@ def _resolve_any_context_variables(
     return processed_parameters, processed_return_value_test
 
 
+def _validate_chain(chain: int):
+    if not isinstance(chain, int):
+        raise ValueError(f'"chain" must be a the integer of a chain ID (got "{chain}").')
```

Comment:
```suggestion
        raise ValueError(f'"chain" must be an integer representing the blockchain ID (got "{chain}").')
```

Reply:
Feels like other than for our internal testing of old tests (since we started off with a non-integer chain id) the check below should be sufficient (L97), and this one is not needed...? Really just a remnant of the old way of doing things. wdyt?

Reply:
I agree, It's mostly just a debugging step that I decided to leave since the highly-specific error message might be helpful if a string is passed.